builtin/settingtypes.txt

```diff
@@ -556,6 +556,26 @@ arm_inertia (Arm inertia) bool true
 #    For example: 0 for no view bobbing; 1.0 for normal; 2.0 for double.
 view_bobbing_amount (View bobbing factor) float 1.0 0.0 7.9
 
+[**Damage]
+
+#   Damage flash, causes the screen to flash reduce
+#   when the player takes damage.
+damage_flash_enable (Damage flash) bool true
```

Comment:
Would there be any particular use-case to have this configurable as a multiplier to weaken or intensify the feedback?  If so: apply the multiplier (likely ranging from 0 to 2) after the `MYMIN` macro.